azure/activity_logs_monitoring/index.js

```diff
@@ -44,6 +44,31 @@ const SCRUBBER_RULE_CONFIGS = {
     // }
 };
 
+/*
+To split array-type fields in your logs into individual logs, you can add sections to the map below. An example of
+a potential use case with azure.datafactory is there to show the format:
+{
+  source_type:
+    path: [list of fields in the log payload to iterate through to find the one to split],
+    keep_original_log: bool, if you'd like to preserve the original log in addition to the split ones or not
+}
+You can also set the DD_LOG_SPLITTING_CONFIG env var with a JSON string in this format.
+*/
+const DD_LOG_SPLITTING_CONFIG = {
+    // 'azure.datafactory': {
```

Comment:
commented out code

Reply:
This is on purpose to give customers the example of what they can change it to